.github/workflows/release.yml

```diff
@@ -480,4 +480,87 @@ jobs:
       with:
         name: signed-debian-package
         path: |
-          signed
\ No newline at end of file
+          signed
+
+# ================================
+#             Publish
+# ================================
+  create-github-release:
+    name: Publish GitHub Draft Release
+    runs-on: ubuntu-latest
+    needs: [ sign-and-notarize-macos, sign-windows, sign-linux ]
+    steps:      
+      - name: Check out repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # Indicate full history so Nerdbank.GitVersioning works.
+
+      - name: Set up dotnet
+        uses: actions/setup-dotnet@v2
+        with:
+          dotnet-version: 6.0.201
+      
+      # Install Nerdbank.GitVersioning
+      - uses: dotnet/nbgv@master
+        with:
+          setCommonVars: true
+      
+      - name: Download artifacts
+        uses: actions/download-artifact@v2
+
+      - uses: actions/github-script@v4
```

Comment:
It's a shame (and surprising!) there's no official GitHub Action for creating a release 😢 

Looks like https://github.com/actions/create-release was deprecated. 😖

Reply:
Yes! I put a little note about why I made this choice in the commit description.

s/o to @vdye who did all the heavy lifting in Microsoft Git - it was a pretty easy copy/paste with a few little tweaks.